packages/frontend/src/components/ProjectConnection/index.tsx

```diff
@@ -299,7 +299,8 @@ export const CreateProjectConnection: FC<CreateProjectConnectionProps> = ({
 }) => {
     const history = useHistory();
     const { user, health } = useApp();
-    const { activeJobIsRunning, activeJob } = useActiveJob();
+    const [createProjectJobId, setCreateProjectJobId] = useState<string>();
+    const { activeJobIsRunning, activeJobId, activeJob } = useActiveJob();
```

Comment:
question (out of the scope of this PR): do we need to convert active job to be a job queue so we can have access to completed/ongoing jobs and its results?